azure/terraform/services/airflow-values.yml

```diff
@@ -0,0 +1,104 @@
+executor: ${airflow_executor}
```

Comment:
Would it be possible to name this file more obviously helm related?

Or we could have a subfolder `services/airflow/values.yml`, so it's not at the top-level of `services`?

Reply:
Yes Will do